TeleportMenu.lua

```diff
@@ -778,6 +786,9 @@ function tpm:CreateSeasonalTeleportFlyout()
 			local flyOutButton = CreateSecureButton(flyOutFrame, "spell", text, spellId)
 			local xOffset = globalWidth * flyoutsCreated
 			flyOutButton:SetPoint("TOPLEFT", flyOutFrame, "TOPLEFT", xOffset, 0)
+			flyOutButton:SetScript("OnMouseDown", function()
```

Comment:
This is alredy set due to the SetScript in the CreateSecureButton() function?

Reply:
You are right; I removed this part and only kept the implementation in CreateSecureButton()